libs/tracing/ppx/ppx_tracing.ml

```diff
@@ -43,6 +43,13 @@ open Ast_helper
 (* Helpers *)
 (*****************************************************************************)
 
+type level = Info | Debug
```

Comment:
Is it better if I just use the strings directly? I thought about options and I think the extra type isn't really necessary given what I'm doing with it

Reply:
See https://github.com/semgrep/semgrep/pull/10121#discussion_r1585345486; I think you can just derive show for the existing type and use that.

Reply:
Unfortunately that doesn't work because `show_level` will output the string `Tracing.Info` but I need to give the ast `LDot (LIdent ("Tracing", "Info"))`

Reply:
sad